Xamarin.Forms.Platform.Android/Renderers/ShellSearchView.cs

```diff
@@ -89,8 +90,10 @@ void ITextWatcher.OnTextChanged(ICharSequence s, int start, int before, int coun
 		AImageButton _clearButton;
 		AImageButton _clearPlaceholderButton;
 		AImageButton _searchButton;
+		//AButton _cancelButton;
```

Comment:
can we delete this?

Reply:
I m not sure if we will want a Cancel button I think @davidortinau 's call is not having one, we could also hide the one in iOS, but I m not a fan of that